.github/workflows/test_docker.yml

```diff
@@ -34,9 +34,10 @@ jobs:
     - name: Test
       run: |
         CONTAINER_ID="$(docker run --rm -v $(pwd):/root/playwright --name playwright-docker-test -d -t playwright-python:localbuild /bin/bash)"
+        docker exec --workdir /root/playwright/ "${CONTAINER_ID}" apt-get install unzip python3-venv -y
```

Comment:
`unzip` is required for updating wheel's `RECORD` file and `python3-venv` is required to run `virtualenv` tests